index.bs

```diff
@@ -4078,30 +4090,12 @@ Workshops and Symposia</h2>
 	This helps ensure that speakers and authors
 	have adequate time to prepare position papers and talks.
 
-<h2 id="Liaisons">
-Liaisons</h2>
-
-	W3C uses the term <dfn export>liaison</dfn>
-	to refer to coordination of activities with a variety of organizations,
-	through a number of mechanisms
-	ranging from very informal
-	(e.g., an individual from another organization participates in a W3C Working Group,
-	or just follows its work)
-	to mutual membership,
-	to even more formal agreements.
-	Liaisons are not meant to substitute for W3C membership.
-
-	All liaisons <em class="rfc2119">must</em> be coordinated by the [=Team=]
```

Comment:
I think we need to keep this part. W3C does receive formal liaisons from other SDOs, and these sometimes do have restrictions relating to confidentiality. I agree this isn't necessarily "all liaisons", but these steps do apply so some of them.